README.md

```diff
@@ -37,13 +48,17 @@ The plugin configuration JSON file will need to reference the Vault plugin as fo
       "implementation": "com.avast.marathon.plugin.vault.VaultPlugin",
       "configuration": {
         "address": "http://address_to_your_vault_instance:port",
-        "token": "access_token"
+        "token": "access_token",
+        "sharedRoot": "secret/shared/",

```

Comment:
What do you think about `absolutePathRoot` and `appRelativePathRoot` instead?

Reply:
it's probably more descriptive. I'll changed it.